docker/sysds.Dockerfile

```diff
@@ -60,14 +60,16 @@ RUN apt-get update -qq \
 	rm -r target/hadoop-test && \
 	rm -r target/maven-archiver && \
 	rm -r target/systemds-** && \
-	rm -r docker && \
+	# rm -r docker && \
```

Comment:
you cannot just comment out a line in this multi-line instruction, then everything after it does not execute.

Reply:
```suggestion
```

Reply:
Not sure what you mean?
just for information the reason why i have this line is to remove the docker folder from the internal copied systemds repository, it does not make a difference for the building of the docker image except it makes the end result slightly smaller.

so therefore, don't out comment it :)